src/components/orderItem.js

```diff
@@ -61,12 +72,16 @@ class OrderItem extends React.Component {
             
          </div>
       </div>
-              
-                
-                
+                     
             </div>
         )
     }
 }
 
-export default connect()(OrderItem);
+
+const stateMappings = state => ({})
+const actionMappings = {
+    dropDown
+}
+
+export default connect(stateMappings, actionMappings)(OrderItem);
```

Comment:
Same thing -- because this is a lower level component (a building block, if you will) you probably don't want to connect it to the redux store. It's best to save those bindings for higher level components like the order component or even the page.

Fine for now but will need to be updated/removed at some point.